outport/host/driver.go

```diff
@@ -102,6 +114,21 @@ func (o *hostDriver) handleAction(args interface{}, topic string) error {
 	return nil
 }
 
+// RegisterHandlerForSettingsRequest will register the handler function for the settings request
+func (o *hostDriver) RegisterHandlerForSettingsRequest(handlerFunction func() error) error {
+	return o.payloadProc.SetHandlerFuncForTopic(handlerFunction, outport.TopicSettings)
+}
+
+// SetCurrentSettings will send the current settings
+func (o *hostDriver) SetCurrentSettings(config outport.OutportConfig) error {
+	configBytes, err := o.marshaller.Marshal(&config)
```

Comment:
We should use `handleAction` to keep consistency within `hostDriver`

Reply:
done